src/main/java/net/sourceforge/tess4j/util/ImageIOHelper.java

```diff
@@ -194,11 +193,11 @@
             }
 
             File tiffFile = File.createTempFile(OUTPUT_FILE_NAME, TIFF_EXT);
```

Comment:
@nguyenq the changes of this file are safe and makes sense for me, though ios is not being closed anymore.